robot_skills/src/robot_skills/test_tools/test_robot.py

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
Missing shebang?

Reply:
Why? The purpose of this is that deriving packages can use this class in their tests

Reply:
I thought this should be executable. But a second look proves me wrong.